console/backend/src/main/java/org/frankframework/management/web/spring/ApiException.java

```diff
@@ -0,0 +1,108 @@
+/*
+   Copyright 2024 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.management.web.spring;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.frankframework.core.IbisException;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.lang.Nullable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ApiException extends RuntimeException implements Serializable {
+
+	private static final long serialVersionUID = 2L;
+	private final transient Logger log = LogManager.getLogger(this);
+	private final HttpStatus status;
```

Comment:
Yes we need to keep track of this, the exception can be created anywhere in the lifecycle of a http request to a response, once thrown it will be picked up by Spring's ExceptionHandler annotation in `ApiEceptionHandler`, only there will the `getResponse` be called where the http status and error response is made

Reply:
Ok, thanks for checking this!